app/src/main/java/de/tum/in/tumcampusapp/activities/SurveyActivity.java

```diff
@@ -48,22 +47,26 @@
 import retrofit.RetrofitError;
 import retrofit.client.Response;
 
-
+/**
+ * SurveyActivity for handling submitting ownQuesitons and reviewing responses
+ */
 public class SurveyActivity extends ProgressActivity {
 
-    Spinner aSpinner1;
-    TextView selectTv;
-    TabHost tabHost;
-    Button submitSurveyButton, facultiesButton;
-    ArrayList<String> questions = new ArrayList<>();
-    ArrayList<String> selectedFaculties = new ArrayList<>();
-    boolean[] checked = new boolean[14];
-    LinearLayout mainResponseLayout, questionsLayout;
-    String chosenFaculties = "", lrzId;
-    ArrayList<String> fetchedFaculties = new ArrayList<>();
-    ViewGroup parentView;
-    String[] numQues = new String[3];
+    private Spinner aSpinner1;
+    private TextView selectTv;
+    private TabHost tabHost;
+    private Button submitSurveyButton, facultiesButton;
+    private ArrayList<String> questions = new ArrayList<>();
+    private ArrayList<String> selectedFaculties = new ArrayList<>();
+    private boolean[] checked = new boolean[14];
+    private LinearLayout mainResponseLayout, questionsLayout;
+    String chosenFaculties = "";
+    private ArrayList<String> fetchedFaculties = new ArrayList<>();
+    private ViewGroup parentView;
+    private String[] numQues = new String[3];
     private SurveyManager surveyManager;
```

Comment:
Please check, if all of these variables really need to be class vars. Maybe you can move some of them to the functions?